sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/IngressGatewayAsset.cs

```diff
@@ -0,0 +1,97 @@
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("\"foo\"")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson(new { key = "value" })</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("{\"key\": \"value\"}")</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// </list>
+        /// </para>
+        /// </summary>
+        private IDictionary<string, BinaryData> _serializedAdditionalRawData;
+
+        /// <summary> Initializes a new instance of <see cref="IngressGatewayAsset"/>. </summary>
+        internal IngressGatewayAsset()
+        {
+        }
+
+        /// <summary> Initializes a new instance of <see cref="IngressGatewayAsset"/>. </summary>
+        /// <param name="hostname"> The ingress hostname. </param>
+        /// <param name="serverId"> The arc ingress gateway server app id. </param>
+        /// <param name="tenantId"> The target resource home tenant id. </param>
+        /// <param name="namespaceName"> The namespace name. </param>
+        /// <param name="namespaceNameSuffix"> The suffix domain name of relay namespace. </param>
+        /// <param name="hybridConnectionName"> Azure Relay hybrid connection name for the resource. </param>
+        /// <param name="accessKey"> Access key for hybrid connection. </param>
+        /// <param name="expiresOn"> The expiration of access key in unix time. </param>
+        /// <param name="serviceConfigurationToken"> The token to access the enabled service. </param>
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        internal IngressGatewayAsset(string hostname, string serverId, Guid? tenantId, string namespaceName, string namespaceNameSuffix, string hybridConnectionName, string accessKey, long? expiresOn, string serviceConfigurationToken, IDictionary<string, BinaryData> serializedAdditionalRawData)
+        {
+            Hostname = hostname;
+            ServerId = serverId;
+            TenantId = tenantId;
+            NamespaceName = namespaceName;
+            NamespaceNameSuffix = namespaceNameSuffix;
+            HybridConnectionName = hybridConnectionName;
+            AccessKey = accessKey;
+            ExpiresOn = expiresOn;
+            ServiceConfigurationToken = serviceConfigurationToken;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> The ingress hostname. </summary>
+        public string Hostname { get; }
+        /// <summary> The arc ingress gateway server app id. </summary>
+        public string ServerId { get; }
```

Comment:
Is it a Guid?

Reply:
Yes it is. Updated with UUID type

Reply:
It's still `string`. Please double check.